src/main/java/ru/yandex/qatools/allure/jenkins/dsl/AllurePluginJobDslExtension.java

```diff
@@ -25,7 +25,7 @@
 import javaposse.jobdsl.plugin.DslExtensionMethod;
 
 /**
- * @author Marat Mavlutov <{@literal mavlyutov@yandex-team.ru}>
+ * @author Marat Mavlutov
```

Comment:
Why remove the email address when it does not need to be removed?

```suggestion
 * @author <a href="mailto:mavlyutov@yandex-team.ru">Marat Mavlutov</a>
```